pkg/proxy/apis/config/validation/validation.go

```diff
@@ -163,14 +163,22 @@ func validateKubeProxyConntrackConfiguration(config kubeproxyconfig.KubeProxyCon
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("Min"), config.Min, "must be greater than or equal to 0"))
 	}
 
-	if config.TCPEstablishedTimeout.Duration < 0 {
+	if config.TCPEstablishedTimeout != nil && config.TCPEstablishedTimeout.Duration < 0 {
```

Comment:
oh, I see, that is why it didn't panic

```
pkg/proxy/apis/config/v1alpha1/defaults.go:     if obj.Conntrack.TCPEstablishedTimeout == nil {
pkg/proxy/apis/config/v1alpha1/defaults.go:             obj.Conntrack.TCPEstablishedTimeout = &metav1.Duration{Duration: 24 * time.Hour} // 1 day (1/5 default)
```

Reply:
yup!
We should have a nil check ... just in case we remove defaulting in the future

Reply:
this one also doesn't panic for the same reason
```
fs.DurationVar(&o.config.Conntrack.TCPEstablishedTimeout.Duration, "conntrack-tcp-timeout-established", o.config.Conntrack.TCPEstablishedTimeout.Duration, "Idle timeout for established TCP connections (0 to leave as-is)")
```